facia-tool/public/js/modules/content-api.js

```diff
@@ -186,13 +195,18 @@ function decorateBatch (articles) {
         }
 
         results.forEach(function(result) {
-            var icc = internalContentCode(result);
+            var pageCode = internalPageCode(result),
+                icc = internalContentCode(result);
 
-            if(icc) {
-                cache.put('contentApi', icc, result);
+            if (pageCode) {
+                cache.put('contentApi', pageCode, result);
+                if (icc) {
```

Comment:
I hate switches, they keep breaking the build. I'm tracking the count in a special page and eventually I'll write a tool to rewrite the old code id with the new one. I won't let this code outlive me.